### DISCUSSION TITLE
Kong OSS 2.8.3 least-connection algorithm

### DISCUSSION THREAD

**Discussion by ankujuniyal** [Kong/kong#11953] [Help] [answered]
Just wanted to know do we have an built in feature in Kong oss 2.8.3 for least-connection algorithm for traffic distribution.
as enterprise Kong started from post version 1.3, so does Kong OSS also have this feature or not.

**Comment by guanlan** [comment, marked as answer]
Kong Gateway OSS supports `least-connections` algorithm, see https://docs.konghq.com/gateway/3.5.x/admin-api/#request-body-22
